src/routes/write-exercise/components/Feedback.tsx

```diff
@@ -59,11 +58,8 @@ const Feedback = ({ documentWithResult, submission, needsToBeRepeated }: Feedbac
   } else {
     Icon = CloseCircleIconBold
     background = BannerRed
-    message = needsToBeRepeated
-      ? `${getLabels().exercises.write.feedback.wrong}\n${wrongWithCorrectSolution}`
-      : wrongWithCorrectSolution
+    message = !submission ? wrongWithCorrectSolution : `${getLabels().exercises.write.feedback.wrong}`
```

Comment:
Why did you remove the check for `needsToBeRepeated`?

Reply:
I thought that it was irrelevant in this exact component, because without including !submission, it was doing nothing.

Reply:
I have included both of them in else block now